gulpfile.js

```diff
@@ -1592,9 +1595,34 @@ gulp.task(
   gulp.series(setTestEnv, "generic", "components", function runBotTest() {
     return streamqueue(
       { objectMode: true },
-      createTestSource("unit", true),
-      createTestSource("font", true),
-      createTestSource("browser (no reftest)", true),
+      createTestSource("unit", { bot: true }),
+      createTestSource("font", { bot: true }),
+      createTestSource("browser", { bot: true }),
+      createTestSource("integration")
+    );
+  })
+);
+
+gulp.task(
+  "xfatest",
+  gulp.series(setTestEnv, "generic", "components", function runXfaTest() {
+    return streamqueue(
+      { objectMode: true },
+      createTestSource("unit"),
+      createTestSource("browser", { xfaOnly: true }),
```

Comment:
Why are we not running the font tests here while we do on the bots?

Reply:
Because this is *basically* a copy of the pre-existing `gulp test` command, and we don't run the `fonttest`s there either :-)

The reason for this, as far as I understand, is that the `fonttest`s requires a lot more (and manual) setup than the rest. While the other test-suites can be run after simply executing a couple of `npm` commands, as outlined in e.g. the README, the `fonttest`s require that you *manually* install both Python and ttx, and that's probably not entirely necessary for most contributors.

Since having a permanently failing test-suite locally (for occasional contributors) probably doesn't look good, and given how few `fonttest`s we actually have, that's probably why it's only run on the bots by default.

Reply:
Good point; I didn't see that the normal test command also didn't run it, but the Python/ttx reason indeed makes sense.